src/insertWord.js

```diff
@@ -23,9 +22,8 @@ async function addWord() {
     return false;
   }
 
-  const word = insertUi.createWordForUser(JSON.parse(localStorage.getItem(CURR_USER_KEY)));
-  wordService.saveWord(word);
-
+  const word = insertUi.createWordForUser(JSON.parse(localStorage.getItem(LOCALSTORAGE_KEY)));
```

Comment:
You have to import `LOCALSTORAGE_KEY` from `Constant.js`. Also, didn't you rename this to `CURR_USER_KEY`? Be consistent.